### PR TITLE
New vc12 and vc14 packages. Remove windows-2016-integration jobs from CI.

### PR DESCRIPTION
**Pull request by JHBoricua**
Signed-off-by: Jose Hernandez <jhboricua@gmail.com>

# Description

Updates the download links and checksums to match latest packages for vc12 and vc14.

## Issues Resolved

Cookbook fails to run due to outdated links/checksums.

## Check List

- [X] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by derekgroh** [comment]
Remove the delivery job from ci.yml should allow the tests to run.

**Comment by JHBoricua** [comment]
@derekgroh Yeah, I wasn't sure if it was still used since it was still in the repo, so I held off on removing it. Will do.

**Comment by JHBoricua** [comment]
@derekgroh Should I also remove the .delivery folder as well?

**Comment by JHBoricua** [comment]
@derekgroh I'm not able to update workflows so I can't remove the delivery job. Must be a permissions thing.

**Comment by JHBoricua** [comment]
@derekgroh I misread the error I was getting, disregard that message about permissions. Jobs are running now.

**Comment by JHBoricua** [comment]
@derekgroh None of the windows 2016 jobs ran and I see this on the job view:
`The windows-2016 environment is deprecated and will be removed on April 1st, 2022.`

Should I delete the 'integration-windows-2016' jobs too?

**Comment by derekgroh** [comment]
Yes, I would vote we remove the windows-2016 integration.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of vcruntime.
 
 ## Unreleased
 
+- Update vc12 and vc14 checksums for new releases of 2013, 2015-2019 and 2015-2022 runtimes [@jhboricua](https://github.com/jhboricua)
+
```
Comment (derekgroh) [inline CHANGELOG.md:8]: Please include that Chef Delivery was removed

Comment (JHBoricua) [reply CHANGELOG.md:10]: Done.

**Review by derekgroh — APPROVED** [review]

**Comment by JHBoricua** [comment]
Can someone with the require privileges merge this, please?

**Comment by JHBoricua** [comment]
Hello. I resolved the merge conflict that was created over the weekend after another PR was merged. This is ready to be reviewed/merged.

**Comment by bmhughes** [comment]
The GH actions CI tests will need fixing before we can merge this, looks like it's still expecting the server 2016 tests to run.

**Comment by JHBoricua** [comment]
> The GH actions CI tests will need fixing before we can merge this, looks like it's still expecting the server 2016 tests to run.

I had already removed the 2016 jobs from the ci.yml file. Is it still expecting them because that change is yet to be merged?

**Comment by JHBoricua** [comment]
Hello. Checking again to see if this PR can be merged.

**Comment by JHBoricua** [comment]
Well, this has been a discouraging experience. I don't know why it's been so difficult to get this PR merged, been trying since the end of April to get someone to pay attention. And I wouldn't be surprised if there has been new releases of VC runtimes by now. So I'm just going to close this, as there seems to be no interest in moving this forward.

**Pull request closed without merging**